src/Modules/Windows-Core/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -24,7 +24,6 @@ CmdletsToExport= "Format-List", "Format-Custom", "Format-Table", "Format-Wide",
     "Unblock-File", "Get-Runspace", "Debug-Runspace", "Enable-RunspaceDebug", "Disable-RunspaceDebug",
     "Get-RunspaceDebug", "Wait-Debugger" , "Get-Uptime", "Get-Verb", "Format-Hex"
 FunctionsToExport= "ConvertFrom-SddlString"
-AliasesToExport= "fhx"

```

Comment:
The same - we now control all the aliases in the tests and can safely export all aliases by default.